corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
I don't remember off the top of my head whether this is true, but i think maybe this model might also cache document access - in which case it'd be good to keep here (and just change the view to reference the domain view). but I could be wrong about that in which case it's fine to remove. @dannyroberts I think has looked at this more closely than I have recently